style.css

```diff
@@ -14,6 +14,48 @@ li {
   list-style: none;
 }
 
```

Comment:
Great job Implementing the project requirements :clap: , you mobile popup modal is perfect, let's make the desktop modal perfect too currently the title of the project is not displayed properly please see :point_down: 
![Screenshot from 2022-11-19 19-37-17](https://user-images.githubusercontent.com/97521557/202855061-a699ebdf-2549-408c-abcf-657065a55b2d.png)

figma template :point_down: 
![image](https://user-images.githubusercontent.com/97521557/202855177-4fcd0431-014a-40b2-a32f-e92935589515.png)

As you know one of the objective for the project is to Understand how to parse a Figma design to create a UI. It will definitely help the look and feel of the website and will increase the overall accessibility :pray: